src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -22,6 +24,7 @@
 import jakarta.servlet.http.HttpServletRequest;
 import java.util.List;
 import java.util.UUID;
+import org.apache.commons.lang3.NotImplementedException;
```

Comment:
```suggestion
import bio.terra.common.exception.FeatureNotImplementedException;
```
There is a Terra specific implementation of this exception that would be consistent with other uses in TDR.